portfolio/src/main/webapp/charts.js

```diff
@@ -0,0 +1,103 @@
+
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawChart);
+
+
+function drawMoodChart(arr) {
+  var data = google.visualization.arrayToDataTable(arr);
+
+  var options = {
+    title: 'Mood Progression',
+    curveType: 'function',
+    legend: { position: 'bottom' }
+  };
+
+  var chart = new google.visualization.LineChart(document.getElementById('curve_chart'));
+
+  chart.draw(data, options);
+}
+
+function drawRelationChart(arr) {
+  var data = google.visualization.arrayToDataTable(arr);
+
+  var options = {
+    title: 'Relationship Progression',
+    curveType: 'function',
+    legend: { position: 'bottom' }
+  };
+
+  var chart = new google.visualization.LineChart(document.getElementById('curve_chart2'));
+
+  chart.draw(data, options);
+}
+
+
+function addChoice(value,name){
+    const params = new URLSearchParams();
+    params.append("value", value);
+    params.append("name",name);
+    fetch('/charts', {method: 'POST', body: params});
+}
+
+function fillCharts(){
+    fetch('/charts');
```

Comment:
Is this code that you'll integrate later?

Reply:
When the html loads, it calls fillCharts(). I put "Fetch('/charts')" at the beginning so when the page loads, the date is updated.